### PR TITLE
sleob billing class created

### PR DESCRIPTION
**Pull request by stephenwaite**
what a SLEOB :)

### CODE REVIEW

**Comment by stephenwaite** [comment]
it's testing well now :)

File: interface/billing/sl_eob_invoice.php [hunk 1]
```diff
@@ -4,37 +4,28 @@
...
- * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
- *
  * @package OpenEMR
  * @author  Rod Roark <rod@sunsetsystems.com>
  * @author  Roberto Vasquez <robertogagliotta@gmail.com>
  * @author  Terry Hill <terry@lillysystems.com>
  * @author  Jerry Padgett <sjpadgett@gmail.com>
- * @link    http://www.open-emr.org
+ * @author  Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2016 Rod Roark <rod@sunsetsystems.com>
```
Comment (bradymiller) [inline interface/billing/sl_eob_invoice.php]: this copyright should have 2005-2016

File: library/invoice_summary.inc.php [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-//
-// This program is free software; you can redistribute it and/or
-// modify it under the terms of the GNU General Public License
-// as published by the Free Software Foundation; either version 2
-// of the License, or (at your option) any later version.
+/**
+ * @package OpenEMR
+ * @author Rod Roark <rod@sunsetsystems.com>
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2009 Rod Roark <rod@sunsetsystems.com>
```
Comment (bradymiller) [inline library/invoice_summary.inc.php]: this copyright should have 2005-2010

File: library/billing/src/SLEOB.php [hunk 3]
```diff
@@ -0,0 +1,263 @@
+<?php
+/**
+ * This class has various billing functions for posting charges and payments.
+ *
+ * @package OpenEMR
+ * @author Rod Roark <rod@sunsetsystems.com>
+ * @author Stephen Waite <stephen.waite@cmsvt.com>
+ * @copyright Copyright (c) 2009 Rod Roark <rod@sunsetsystems.com>
```
Comment (bradymiller) [inline library/billing/src/SLEOB.php]: This copyright should be 2005-2009

File: interface/billing/new_payment.php [hunk 4]
```diff
@@ -26,6 +25,7 @@
 require_once("$srcdir/payment.inc.php");
 
 use OpenEMR\Billing\ParseERA;
+use OpenEMR\Billing\SLEOB;
```
Comment (bradymiller) [inline interface/billing/new_payment.php]: Can remove above since script is not using it

File: interface/patient_file/deleter.php [hunk 5]
```diff
@@ -19,9 +19,9 @@
 require_once('../globals.php');
 require_once($GLOBALS['srcdir'].'/log.inc');
 require_once($GLOBALS['srcdir'].'/acl.inc');
-require_once($GLOBALS['srcdir'].'/sl_eob.inc.php');
 
 use OpenEMR\Billing\BillingUtilities;
+use OpenEMR\Billing\SLEOB;
```
Comment (bradymiller) [inline interface/patient_file/deleter.php]: Can remove above since script is not using the class

File: interface/patient_file/front_payment.php [hunk 6]
```diff
@@ -16,13 +16,13 @@
...
 require_once("$srcdir/payment.inc.php");
 require_once("$srcdir/forms.inc");
-require_once("$srcdir/sl_eob.inc.php");
 require_once("$srcdir/invoice_summary.inc.php");
 require_once("../../custom/code_types.inc.php");
 require_once("$srcdir/options.inc.php");
 require_once("$srcdir/encounter_events.inc.php");
 
 use OpenEMR\Billing\BillingUtilities;
+use OpenEMR\Billing\SLEOB;
```
Comment (bradymiller) [inline interface/patient_file/front_payment.php]: Can remove above since script is not using the class

File: portal/portal_payment.php [hunk 7]
```diff
@@ -54,6 +53,7 @@
 require_once("$srcdir/crypto.php");
 
 use OpenEMR\Billing\BillingUtilities;
+use OpenEMR\Billing\SLEOB;
```
Comment (bradymiller) [inline portal/portal_payment.php]: Can remove above since script is not using the class

**Comment by bradymiller** [comment]
Just noted a couple minor things to address.

**Comment by bradymiller** [comment]
hi @stephenwaite , looks good and bringing into codebase. thanks for the new class! -brady

**Pull request merged**